wallet-new/src/Cardano/Wallet/WalletLayer/Types.hs

```diff
@@ -120,6 +123,23 @@ instance Buildable GetWalletError where
     build (GetWalletWalletIdDecodingFailed txt) =
         bprint ("GetWalletWalletIdDecodingFailed " % build) txt
 
+data GetUtxosError =
+      GetWalletUtxosWalletIdDecodingFailed Text
+    | GetUtxosErrorFromGetAccountsError GetAccountsError
+    deriving Eq
+
+instance Show GetUtxosError where
+    show = formatToString build
```

Comment:
Not sure here, but do we actually use `show` to display our errors :s ? 
Aren't we using their `Buildable` instances? If not, then I'd open a ticket to change that, and wouldn't define any custom instance for `Show`.

Reply:
I think the `Show` instances get used in some places (and are required when throwing these as expections). We do the same in a few places. I think the correct solution is to have `Show` be auto-derived and then define `displayException` in terms of the `Buildable` instances, but typically `displayException` is not used in top-level exception handlers. Either way, I'd say open a separate ticket about this because this is not just this PR.

Reply:
@edsko Yes, that's exactly what I meant and I am not expecting such refactoring to happen in this PR :/ ... It is rather bad if we indeed rely on `show` to display exceptions in the top-level handlers. 

I think it would be fine to go directly for `formatToString build` in those handlers (since the actual purpose of the `Buildable` is to construct a user-friendly readable output) yet I don't mind having that exposed as a clear `displayException` :+1: